DictionaryConsoleApp/Program.cs

```diff
@@ -0,0 +1,152 @@
+﻿namespace DictionaryConsoleApp
+{
+    public class Program
+    {
+        private static void Main()
+        {
+            //Exercise1();
+            Exercise2();
+            //Exercise3();
+            //Exercise4();
+        }
+
+        public static void Exercise1()
+        {
+            Console.WriteLine("Exercise 1.");
+
+            var employees = new Dictionary<Student, int>
+            {
+                { new Student(4, "John"), 1},
+                { new Student(5, "Henry"), 2 }
+            };
+
+            foreach (KeyValuePair<Student, int> pair in employees)
+            {
+                Console.WriteLine($"Student {pair.Key.Name}, estimation: {pair.Key.AverageRating}.");
+            }
+        }
+
+        public static void Exercise2()
+        {
+            var dictionaryEnglishFranch = new Dictionary<string, string>(10)
+            {
+                    { "book", "livre" },
+                    { "mouse", "souris" },
+                    { "dictionary", "dictionnaire" },
+                    { "notepad", "bloc-notes" },
+                    { "word", "mot" },
+                    { "time", "temps" },
+                    { "music", "musique" },
+                    { "game", "jeu" },
+                    { "flower", "fleur" },
+                    { "candles", "bougies" }
+            };
+
+            var input = Console.ReadLine();
+
+            Console.WriteLine(GetTranslation(dictionaryEnglishFranch, input));
```

Comment:
https://github.com/DariaDostoevskaia/Exercises/pull/7#discussion_r1357345604
verification was necessary, but different.
Look at what methods String has that check the string for nal and other empty characters